cmd/criticize/main.go

```diff
@@ -153,10 +154,16 @@ func (l *linter) CheckPackage(pkgPath string) {
 	l.ctx.TypesInfo = &pkgInfo.Info
 	l.ctx.Package = pkgInfo.Pkg
 	for _, f := range pkgInfo.Files {
-		l.checkFile(f)
+		l.checkFile(l.getFilename(f), f)
 	}
 }
 
+func (l *linter) getFilename(f *ast.File) string {
+	fname := l.prog.Fset.Position(f.Pos()).String() // ex: /usr/go/src/pkg/main.go:1:1
```

Comment:
Could you reference https://github.com/golang/go/issues/24498 in a comment inside `getFilename` function?